packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/name.assignment.scope

```diff
@@ -0,0 +1,24 @@
+const name = "Hello world";
+---
+
+[Content]
+ const name = "Hello world";
+       ^^^^
+
+[Removal]
+ const name = "Hello world";
+ ^^^^^^^^^^^^^
+
+[Leading delimiter: Content]
```

Comment:
nit: I would rather organize like this:

```
[content]
[content leading]
[content trailing]
[removal]
[removal leading]
[removal trailing]
[domain]
```

etc

I think it is easier to reason about.

Reply:
I don't think I agree. We now group the headers for identical ranges together. The content removal and domain is likely to be the same while the leading and trailing will never. Content and removal also belongs to a single target while leading and trailing is two separate targets.

Reply:
I agrée with @AndreasArvidsson here. Leading content and leading removal are attributes of a single target: the leading target of the primary target here

Tbh I'd be tempted to drop leasing removal and trailing removal unless they differ from leading content / trailing content, because they're almost always the same and their very existence is confusing, and basically only useful for the block scope

Reply:
I mean we do already group the headers if they are identical?

Reply:
update: let's just call it `Leading delimiter` if content and removal range are equal

Reply:
done